p/scripts/main.js

```diff
@@ -1286,7 +1276,59 @@ function init_crypto_form() {
 }
 //</crypto form (Web login)>
 
+var sidebar = document.getElementById('sidebar');
+var useJsScrollbar = true;
+try {
+	/*jshint -W018 */
+	useJsScrollbar = sidebar && !CSS.supports('scrollbar-color: auto') &&
+		!(parseInt(getComputedStyle(sidebar, '::-webkit-scrollbar').width) < sidebar.offsetWidth);
+	/*jshint +W018 */
+} catch (ex) {
+}
+if (useJsScrollbar) {
+	inject_script('simple-scrollbar.min.js');
+}
+
+function sticky_recalc() {
```

Comment:
Maybe not for this time, but I am wondering whether this function could be skipped in the case of native scrollbars.
The tree container is supposed to grow by itself when expanding categories, isn't it?

Reply:
If the `#nav_entries` element was reworked some, it could be. sticky_recalc takes into account the height of that element, which isn't in side .tree, when determining the height; if we left sticky_recalc out on native and left them where they were, the bottom of the scrollbar would extend behind that element. We could do something like the search bar, where one would be inside .tree and displayed on desktop, and another set would be displayed on mobile.

Reply:
Ok. Let's come back to it at a later point

Reply:
Attempt in https://github.com/FreshRSS/FreshRSS/pull/2181